### ISSUE TITLE
[suggestion] only inject livereload script on dev env

### ISSUE THREAD

**Issue by BenGamma** [FNGR2911/murby#6]
Hello !

Here is a suggestion to avoid to inject the [livereload script](https://github.com/FNGR2911/murby/blob/2e8b2c0a8027903c525886b4a308e3abab7f414e/web/site/snippets/footer.php#L2) in the footer even on a production environement.

I'm setting a env variable in my Apache conf file `httpd.conf` :
```
SetEnv DEV_ENV true
```

And in the footer I wrapped the script inject with a condifition :
```php
<?php
    $env = getenv('DEV_ENV', true) ?: getenv('DEV_ENV');
    if($env == true):?>
    <script src="http://localhost:35729/livereload.js"></script>
  <?php endif ?>
```

Let me know what you think, I could make a PR if needed :)

**Comment by FNGR2911** [comment]
Hi @BenGamma! Thank you for that suggestion. It's a good idea, just to print the live reload script in development. So I thought about it and I think the way to use the `httpd.conf` is a little bit to unwieldy. So I just added the ENV via the npm script like that:

```json
  "scripts": {
    "server": "APP_ENV=dev php -S localhost:8282 -t web web/kirby/router.php",
  },
```

Now in the footer template it'll be checked like you summoned:

```php
  <?php if (getenv('APP_ENV') === "dev"): ?>
    <script src="http://localhost:35729/livereload.js"></script>
  <?php endif;?>
```

I updated murby! What du you think about it? :)
